src/app/Dashboard/AddCard.tsx

```diff
@@ -94,47 +114,74 @@ export const AddCard: React.FunctionComponent<AddCardProps> = (props: AddCardPro
 
   const handleAdd = React.useCallback(() => {
     setShowWizard(false);
-    dispatch(addCardIntent(getConfigByTitle(selection).component.name));
-  }, [dispatch, selection]);
+    const cardTitle = getConfigByTitle(selection).component.name;
+    dispatch(addCardIntent(cardTitle, propsConfig));
+  }, [setShowWizard, dispatch, addCardIntent, selection, propsConfig]);
 
   const handleStart = React.useCallback(() => {
     setShowWizard(true);
   }, [setShowWizard]);
 
   const handleStop = React.useCallback(() => {
+    setSelection('');
     setShowWizard(false);
-  }, [setShowWizard]);
+  }, [setSelection, setShowWizard]);
+
+  // custom nav for disabling subsequent steps (ex. configuration) if a card type hasn't been selected first
+  const customNav: CustomWizardNavFunction = React.useCallback(
+    (
+      isExpanded: boolean,
+      steps: WizardControlStep[],
+      activeStep: WizardControlStep,
+      goToStepByIndex: (index: number) => void
+    ) => {
+      return (
+        <WizardNav isExpanded={isExpanded}>
+          {steps.map((step, idx) => (
+            <WizardNavItem
+              key={step.id}
+              id={step.id}
+              content={step.name}
+              isCurrent={activeStep.id === step.id}
+              isDisabled={step.isDisabled || (idx > 0 && !selection)}
+              stepIndex={step.index}
+              onNavItemClick={goToStepByIndex}
+            />
+          ))}
+        </WizardNav>
+      );
+    },
+    [selection]
+  );
 
   return (
     <>
       <Card isRounded isLarge>
         {showWizard ? (
-          <Wizard onClose={handleStop} onSave={handleAdd} height={300}>
-            <WizardStep
-              id="card-type-select"
-              name="Card Type"
-              footer={{ nextButtonText: 'Finish', isNextDisabled: !selection }}
-            >
+          <Wizard isStepVisitRequired onClose={handleStop} onSave={handleAdd} height={500} nav={customNav}>
```

Comment:
I think we can remove `isStepVisitRequired`. This allows the Configuration (Step 2) to be enabled if a valid card type is selected. Placeholder selection will disable this step 2.